explore/src/main/scala/explore/Routing.scala

```diff
@@ -86,14 +86,17 @@ object Routing {
         )
         .onPostRenderP {
           case (prev, next, view)
-              if next.some =!= prev &&
+              if prev.exists(_ =!= next) &&
```

Comment:
If we do `prev.forAll(_ =!= next)` it will also match in case of `prev === None` and we only need one case clause.

Reply:
👍